beets/ui/commands.py

```diff
@@ -872,11 +872,20 @@ def resolve_duplicate(self, task, found_duplicates):
                     list(duplicate.items()) if task.is_album else [duplicate],
                     not task.is_album,
                 ))
+                if config['import']['duplicate_verbose_prompt']:
+                    if not task.is_album:
+                        print(f"  {duplicate}")
+                    else:
+                        for dup in duplicate.items():
+                            print(f"  {dup}")
```

Comment:
This might be slightly easier to read if it were reversed! (That is, start with `if task.is_album:`.)

Reply:
Hahaha, I had it the other way round and thought: That looks badly readable and switched it around. But actually I agree with you, starting with a negated condition when there is an else branch anyway, is often confusing. I'll change it back! :-)